style.css

```diff
@@ -2,579 +2,1338 @@
+    background:
+      linear-gradient(179.35deg, rgba(38, 38, 38, 0) 0.85%, rgba(38, 38, 38, 0.9) 84%),
+      url(images/Snapshoot\ Portfolio-desk4.png);
+    width: 369px;
+    height: 435px;
+    margin-top: 37px;
+    margin-left: 24px;
+  }
+
+  .desktopcard-works5 h1 {
+    width: 337px;
+    height: 81px;
+    font-family: 'Crete Round', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 32px;
+    line-height: 44px;
+    color: #fff;
+    margin: 178px 16px 0 16px;
+  }
+
+  .desktopcard-works5 p {
+    width: 337px;
+    height: 72px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 17px;
+    line-height: 24px;
+    color: #fff;
+    margin: 24px 16px 0 16px;
+  }
+
+  .desktopcard-works5 ul {
+    list-style: none;
+    display: flex;
+    justify-content: flex-start;
+    padding: 0;
+    margin: 24px 0 0 16px;
+  }
+
+  .desktopcard-works5 li {
+    margin-right: 12px;
+    padding: 10px 12px 10px 12px;
+    background: rgba(255, 255, 255, 0.24);
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #fff;
+  }
+
+  .desktopcard-works6 {
+    background:
+      linear-gradient(179.35deg, rgba(38, 38, 38, 0) 0.85%, rgba(38, 38, 38, 0.9) 84%),
+      url(images/Snapshoot\ Portfolio-desk1.png);
+    width: 369px;
+    height: 435px;
+    margin-top: 37px;
+    margin-left: 24px;
+  }
+
+  .desktopcard-works6 h1 {
+    width: 337px;
+    height: 81px;
+    font-family: 'Crete Round', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 32px;
+    line-height: 44px;
+    color: #fff;
+    margin: 178px 16px 0 16px;
+  }
+
+  .desktopcard-works6 p {
+    width: 337px;
+    height: 72px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 17px;
+    line-height: 24px;
+    color: #fff;
+    margin: 24px 16px 0 16px;
+  }
+
+  .desktopcard-works6 ul {
+    list-style: none;
+    display: flex;
+    justify-content: flex-start;
+    padding: 0;
+    margin: 24px 0 0 16px;
+  }
+
+  .desktopcard-works6 li {
+    margin-right: 12px;
+    padding: 10px 12px 10px 12px;
+    background: rgba(255, 255, 255, 0.24);
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #fff;
+  }
+
+  .desktop-about-me {
+    display: flex;
+    flex-direction: column;
+    background-image: url(images/illustration-about\ me@2x.png);
+    background-color: #262626;
+    background-repeat: no-repeat;
+    background-size: 30% auto;
+    background-position: right;
+    margin-top: 130px;
+  }
+
+  .desktop-about-me h1 {
+    margin: 130px 0 0 142px;
+    padding-top: 130px;
+    font-family: 'Crete Round', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 72px;
+    line-height: 82px;
+    color: #f55800;
+  }
+
```

Comment:
please try as much as possible to match your design with the figma since there are some few disparities as shown below. 

 Your design            |  The design template
:-------------------------:|:-------------------------:
![Screenshot from 2022-11-12 12-21-06](https://user-images.githubusercontent.com/65233860/201474313-a9644ebb-2a4c-4d1b-a794-19cc7366d5b3.png)  |  ![Screenshot from 2022-11-12 12-19-40](https://user-images.githubusercontent.com/65233860/201474332-921750d8-6097-48bd-8550-b505d361c946.png)



 Your design            |  The design template
:-------------------------:|:-------------------------:
![Screenshot from 2022-11-12 12-21-27](https://user-images.githubusercontent.com/65233860/201474348-a779cd50-ecc6-44aa-8aeb-292ae0c98982.png)  |  ![Screenshot from 2022-11-12 12-20-19](https://user-images.githubusercontent.com/65233860/201474363-b2798e44-71dd-41f0-a297-2c23f6b82282.png)